Gifski/AppDelegate.swift

```diff
@@ -297,16 +280,32 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 		circularProgress.progress = 0
 		circularProgress.animated = true
 
-		progress = Progress(totalUnitCount: 1)
-		progress?.becomeCurrent(withPendingUnitCount: 1)
-		gifski = Gifski.convertFile(
-			inputUrl,
-			outputUrl: outputUrl,
-			quality: defaults["outputQuality"] as! Double,
+		let progress = Progress(totalUnitCount: 1)
+		progress.becomeCurrent(withPendingUnitCount: 1)
+		Gifski.convert(
+			fileAt: inputUrl,
+			outputTo: outputUrl,
+			withQuality: defaults["outputQuality"] as! Double,
 			dimensions: choosenDimensions,
 			frameRate: choosenFrameRate
-		)
-		progress?.resignCurrent()
+		) { [weak self] result in
```

Comment:
No I don't think it's needed in this case. I just like to add it just in case I copy paste code into a view controller in the future. I can remove it tho!